legal-api/src/legal_api/resources/business/business_tasks.py

```diff
@@ -61,7 +61,10 @@ def get(identifier):
     def construct_task_list(business):
         tasks = []
         order = 1
-        todo_start_date = datetime(2019,1,1)  # If no filings exist in legal API db this year will be used as the start year.
+        checkAgm = validations.annual_report.RequiresAGM(business)
```

Comment:
maybe rename the variable name to 'hasAgm' or something that clearly identifies it as a boolean? Super minor tho.